framework/android/src/main/cpp/minimum_exports.lst

```diff
@@ -25,6 +25,8 @@
   global:
     JNI_OnLoad;
     JNI_OnUnload;
+    Java_com_tencent_tdf_keyboard_KeyboardChannel_*;
```

Comment:
Using JNI dynamic registration instead of static exposure

Reply:
fixed